tools/serverpod_cli/bin/generator/class_analyzer.dart

```diff
@@ -523,3 +608,22 @@ class ClassAnalyzer {
     return true;
   }
 }
+
+extension on TypeDefinition {
+  TypeDefinition detectProtocolReferences(
```

Comment:
So first I parse the protocol definition, then I try to auto detect `protocol:` prefixes in types. So whenever no url is set and I know the user specified a class/enum with the same symbol name. I default to the `protocol:` prefix.

This makes the `protocol:` prefix optional.

Reply:
I was mostly referring to renaming the the method to something like `applyProtocolReferences` as it would be more in line with what the method does. :)